tests/register/register.go

```diff
@@ -31,6 +38,68 @@ func init() {
 	Tests = make(map[TestType][]types.Test)
 }
 
+var count = 0
+
 func Register(tType TestType, t types.Test) {
```

Comment:
Is there any reason why a test should be bypassing the `RegisterTest` function of generating multiple tests? If not we should rename this to `register` and rename `RegisterTest` to just `Register`. Then all the individual tests do not need to be bumped.